cypress/features/regression/experimental/textboxValidations.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Textbox validations component
+    I want to change Textbox validations component  properties
+
+    Background: Open Textbox validations component page
+        Given I open "Experimental Textbox" component page validations
+
+    @positive
+    Scenario Outline: Verify the <state> validation of Textbox component
+        Given Type "<keyWord>" text into input and select the value
```

Comment:
we don't need to do ` and select the value` because it's only for selecting dropdown element in `Select` component

Reply:
fixed